docs/sources/auth/azuread.md

```diff
@@ -0,0 +1,103 @@
++++
+title = "Azure AD OAuth2 Authentication"
+description = "Grafana OAuthentication Guide "
+keywords = ["grafana", "configuration", "documentation", "oauth"]
+type = "docs"
+[menu.docs]
+name = "Azure AD"
+identifier = "azuread_oauth2"
+parent = "authentication"
+weight = 3
++++
+
+# Azure AD OAuth2 Authentication
```

Comment:
Per the Documentation Style Guide, all headings should be in sentence case. 

Also, this topic looks like a task, so I suggest starting it with a verb. Something like Set up Azure AD OAuth2 authentication.